openlibrary/templates/search/sort_options.html

```diff
@@ -19,6 +19,7 @@
        sort_options = [
          { 'sort': 'relevance', 'name': _("Relevance"), 'ga_key': 'Relevance' },
          { 'sort': 'editions', 'name': _("Most Editions"), 'ga_key': 'Editions' },
+         { 'sort': 'title', 'name': _("Alphabetical"), 'ga_key': 'Alphabetical' },
```

Comment:
Let's do this please and update the code correspondingly!

Also since the name is now rather long, let's add this to the end.

```suggestion
         { 'sort': 'title', 'name': _("Work Title (beta: Librarian only)"), 'ga_key': 'Title', 'show': ctx.user and (ctx.user.is_admin() or ctx.user.is_usergroup_member('/usergroup/librarians'))  },
```